core/drivers/imx_i2c.c

```diff
@@ -406,15 +424,15 @@ static TEE_Result i2c_init(void)
 {
 	size_t n = 0;
 
-	if (get_va(i2c_clk.base.pa, &i2c_clk.base.va) != TEE_SUCCESS)
+	if (get_va(i2c_clk.base.pa, &i2c_clk.base.va))
 		return TEE_ERROR_GENERIC;
 
-	if (get_va(i2c_mux.base.pa, &i2c_mux.base.va) != TEE_SUCCESS)
+	if (get_va(i2c_mux.base.pa, &i2c_mux.base.va))
 		return TEE_ERROR_GENERIC;
 
 	for (n = 0; n < ARRAY_SIZE(i2c_bus); n++) {
-		if (get_va(i2c_bus[n].pa, &i2c_bus[n].va) != TEE_SUCCESS)
-			return TEE_ERROR_GENERIC;
+		if (get_va(i2c_bus[n].pa, &i2c_bus[n].va))
```

Comment:
No real need to remove `!= TEE_SUCCESS` here and above.

Reply:
right, but since I need to maintain this file I'd like to do some minor tidy up on this